latex/delegation.tex

```diff
@@ -253,6 +311,46 @@
   \label{fig:delegation-rules}
 \end{figure}
 
+
+The rules for registering and delegating stake keys are given in
+\cref{fig:delegation-rules}. The preconditions for the registration of a stake
+key ensure that a given certificate $c$ is of the correct type
+(i.e. $\DCertRegKey$),
+and that the hash key associated with the author of the certificate is not
+already found in the current list of stake keys.
+
+In order to delegate to a stake pool, a stake key must first be registered.
+When registering a new stake key (the Deleg-Reg inference rule),
```

Comment:
```suggestion
When registering a new stake key (the \cref{eq:deleg-reg} inference rule),
```

Reply:
I changed all the rule names to references